docs/source/markdown/podman-systemd.unit.5.md

```diff
@@ -159,44 +149,33 @@ If enabled, makes image read-only, with /var/tmp, /tmp and /run a tmpfs (unless
-starting at `RemapUidStart` (and gids starting at `RemapGidStart`) in the container are chosen from the
-available host uids specified by `RemapUidRanges` (and `RemapGidRanges`).
-
-#### `RemapUidStart=` (defaults to `1`)
-
-If `RemapUsers` is enabled, this is the first uid that is remapped, and all lower uids are mapped
-to the equivalent host uid. This defaults to 1 so that the host root uid is in the container, because
-this means a lot less file ownership remapping in the container image.
-
-#### `RemapGidStart=` (defaults to `1`)
-
-If `RemapUsers` is enabled, this is the first gid that is remapped, and all lower gids are mapped
-to the equivalent host gid. This defaults to 1 so that the host root gid is in the container, because
-this means a lot less file ownership remapping in the container image.
+#### `RemapUsers=`
 
-#### `RemapUidRanges=`
+If this is set, then host user and group ids are remapped in the container. It currently
+supports values: `auto`, `manual` and `keep-id`.
 
-This specifies a comma-separated list of ranges (like `10000-20000,40000-50000`) of available host
-uids to use to remap container uids in `RemapUsers`. Alternatively, it can be a username, which means
-the available subuids of that user will be used.
+In `manual` mode, the `RemapUid` and `RemapGid` options can define an
+exact mapping of uids from host to container. You must specify these.
 
-If not specified, the default ranges are chosen as the subuids of the `quadlet` user.
+In `auto` mode mode, the subuids and subgids allocated to the `containers`user is used to allocate
+host uids/gids to use for the container. By default this will try to estimate a count of the ids
+to remap, but you can set RemapUidSize to use an explicit size. You can also Use `RemapUid` and
+`RemapGid` key to force a particular host uid to be mapped to the container.
 
-#### `RemapGidRanges=`
+In `keep-id` mode, the running user is mapped to the same id in the container. This is supported
+only on user systemd units.
 
-This specifies a comma-separated list of ranges (like `10000-20000,40000-50000`) of available host
-gids to use to remap container gids in `RemapUsers`. Alternatively, it can be a username, which means
-the available subgids of that user will be used.
+#### `RemapUid=`
 
-If not specified, the default ranges are chosen as the subgids of the `quadlet` user.
+If `RemapUsers` is enabled, this specifies a uid mapping of the form `container_uid:from_uid:amount`,
+which will map `amount` number of uids on the host starting at `from_uid` into the container, starting
+at `container_uid`.
 
-#### `KeepId=` (defaults to `no`, only works for user units)
+#### `RemapGid=`
 
-If this is enabled, then the user uid will be mapped to itself in the container, otherwise it is
-mapped to root. This is ignored for system units.
+If `RemapUsers` is enabled, this specifies a gid mapping of the form `container_gid:from_gid:amount`,
```

Comment:
s/RemapUsers/RemapGid/?

Reply:
No, you need to set RemapUsers to some mode for this to work.

Reply:
ok